css-align-3/resources/alignment-parsing-utils.js

```diff
@@ -0,0 +1,19 @@
+var position = [ "start", "end", "self-start", "self-end", "left", "right", "center", "flex-start", "flex-end"];
+var distribution = [ "stretch", "space-around", "space-between", "space-evenly"];
+var baseline = [ "baseline", "last-baseline"];
```

Comment:
I know these properties will be only used on alignment tests, but still I prefer a more explicit name like: `positionValues`, `distributionValues` and `baselineValues`.